app/src/main/java/com/example/triptracker/model/itinerary/ItineraryList.kt

```diff
@@ -17,6 +20,13 @@ data class ItineraryList(var itineraryList: List<Itinerary>) {
     return itineraryList.filter { it.title.contains(task, ignoreCase = true) }
   }
 
+  // Returns filtered itinerary based on LatLngBounds of the screen
```

Comment:
We could change the comment type to /** */ to have a uniform way of commenting functions